narayana/ArjunaJTA/jta/tests/classes/org/jboss/narayana/rts/TxnHelper.java

```diff
@@ -51,7 +49,7 @@ public static Set<Link> beginTxn(Client client, String txurl) throws IOException
                     Entity.entity(new Form(), MediaType.APPLICATION_FORM_URLENCODED_TYPE));
             Set<Link> links = response.getLinks();
 
-            EntityUtils.consume((HttpEntity) response.getEntity());
+            response.getEntity();
```

Comment:
We want to be certain that the entity input stream is fully consumed so you need `response.readEntity(String.class)`  here.

Reply:
Thanks, fixed it.